druid-shell/src/backend/mac/application.rs

```diff
@@ -111,7 +111,8 @@ impl Application {
             let locale: id = msg_send![nslocale_class, currentLocale];
             let ident: id = msg_send![locale, localeIdentifier];
             let mut locale = util::from_nsstring(ident);
-            if let Some(idx) = locale.chars().position(|c| c == '@') {
+            // This is done because the locale parsing library we use expects an unicode locale, but these vars have an ISO locale
+            if let Some(idx) = locale.chars().position(|c| c == '.' || c == '@') {
```

Comment:
I believe this is okay as-is, because macOS doesn't include encoding in the locale?

Reply:
I couldnt find any documentation on this. Having the code be the same across the backends seems nice to me. It hasnt caused any problems so far so it should indeed be fine without.

Reply:
I left the comment in since I think it still applies and is helpful